hazelcast/src/main/java/com/hazelcast/jet/config/InstanceConfigDelegate.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.config;
+
+import javax.annotation.Nonnull;
+
+/**
+ * This class is used to delegate the deprecated {@link InstanceConfig} method
+ * calls to {@link JetConfig}.
+ */
+public final class InstanceConfigDelegate extends InstanceConfig {
```

Comment:
This class should be in another package, the current package is public API and users shouldn't use it directly.

Reply:
The class isn't "a delegate". A delegate is something to which something is delegated. A better name might be `DelegatingInstanceConfig`, the `jetConfig` field then could be named `delegate`, though `jetConfig` is also ok.

Reply:
👍